manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -185,6 +189,7 @@ properties:
     status:
       user: router_user
       password: (( grab secrets.router_password ))
+      port: (( grab meta.gorouter.healthcheck_port ))
```

Comment:
I think it would be cleaner to set the value here, and have the haproxy property grab it from here with `grab properties.router.status.port` instead of adding a meta entry.

Reply:
Good idea, updated.